client/flutter/lib/pages/symptom_checker/question_pages/previous_next_buttons.dart

```diff
@@ -21,9 +21,11 @@ class NextButton extends StatelessWidget {
       // TODO: Localize
       "Next",
       enableNext ? onNext : null,
-      borderRadius: 50,
       crossAxisAlignment: CrossAxisAlignment.center,
-      titleStyle: ThemedText.styleForVariant(TypographyVariant.h4)
+      mainAxisAlignment: MainAxisAlignment.center,
+      verticalPadding: 12,
+      borderRadius: 500,
```

Comment:
sanity-checking the 500 here

Reply:
Yes, it means its curved no matter how big it gets.